### PR TITLE
Add a11y friendly default text-colors when selecting a background-color.

### PR DESCRIPTION
**Pull request by allancole**
**Before:**

**After:**

#### Related Tickets 

Fixes: #964

### CODE REVIEW

**Comment by joyously** [comment]
Don't the color settings change in the palette when you change the colors in the Customizer?
So setting the text color is only good for the defaults?
Wouldn't it be better to make the text color work regardless of background color?

**Comment by allancole** [comment]
Group Blocks and any block that uses any of the `.has-background-xxxxxxx-color` utility classes are also fixed with this PR. Here’s a gif of it working with this fix:

**Comment by mapk** [comment]
I just tested this in FF 70.0.1 and am seeing the a11y warning still. Is this right?

contrast 2019-11-11 14_32_50

**Comment by mapk** [comment]
So double checking the contrast on these color combinations:

- RED (#cd2653) & WHITE (#FFF) – 5.25 contrast ratio
- BLACK (#000) & TAN (#f5efe0) – 18.30 contrast ratio
- DARK GRAY (#6d6d6d) & WHITE (#FFF) – 5.17 contrast ratio

Looks like something may be off with the contrast warning in Gutenberg?

**Comment by joyously** [comment]
I tried it with changed colors.
Peek 2019-11-11 16-48

File: style-rtl.css [hunk 1]
```diff
@@ -2664,6 +2666,7 @@ h2.entry-title {
 
 :root .has-secondary-background-color {
 	background-color: #6d6d6d;
+	color: #FFF;
```
Comment (audrasjb) [inline style-rtl.css]: Coding standards: should be `#fff` for consistency.

**Comment by audrasjb** [comment]
Hi,
For what it worth, I tested the PR on my side, and it looks good to me.
Changing background color is automatically changing the font color when needed.

**Comment by audrasjb** [comment]
5e7ff2db531ab3c9150268adb25fc914

**Pull request merged**